spec/workers/sendgrid_message_worker_spec.rb

```diff
@@ -76,7 +76,7 @@
 
         expect do
           sendgrid_vo = nil
-          Sidekiq::Testing.inline! { SendSendgridMessageWorker.perform_async(sendgrid_vo) }
+          Sidekiq::Testing.inline! { SendgridMessageWorker.perform_async(sendgrid_vo) }
```

Comment:
why haven't we just configured sidekiq to do inline-everything in the test environment?

Reply:
because I was unaware that we could do that.